web/themes/custom/pineapple/templates/user/user--account--full.html.twig

```diff
@@ -18,28 +18,88 @@
 #}
 <article{{ attributes }}>
```

Comment:
A general comment on this file, it feels very crowded can you please add some line breaks between separate components and major divs.

Reply:
Okay, let me know if this is better.